projects/igniteui-angular/src/lib/directives/date-time-editor/date-time-editor.directive.ts

```diff
@@ -561,6 +578,12 @@ export class IgxDateTimeEditorDirective extends IgxMaskDirective implements OnCh
     return newDate;
   }
 
+  private trySpinValue(datePart: DatePart, delta: number, negative = false) {
+    const spinDelta = delta !== 1 ? delta : this.datePartDeltas[datePart] || 1;
```

Comment:
Erm something's off with this check. If the spin value _is_ 1, we ignore it and take the one from the option?